views/editpost.ejs

```diff
@@ -0,0 +1,131 @@
+<!--
+    2021.3.6 16:38 EST Front-end team Leo Li
+    1. Design the single post page
+    2. Add temporary ejs statement in the form of < post.data >... because if i write the normal form (with %), ejs will throw an error.
+
+-->
+
+<body>
+<!-- HEADER CODE -->
+
+<!-- Heading and Description of Forum Page -->
+<%- include('partials/forum-subheader') %>
+<div class="toast" role="alert" aria-live="assertive" aria-atomic="true" data-delay=2000 stye="positon:absolute;">
+    <div class="toast-header">
+        <strong class="mr-auto" id="toasttitle"></strong>
+        <button type="button" class="ml-2 mb-1 close" data-dismiss="toast" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+        </button>
+    </div>
+    <div class="toast-body"></div>
+</div>
+<!-- Main Quill library -->
+<script src="//cdn.quilljs.com/1.3.6/quill.js"></script>
```

Comment:
Remove references to Quill.js since it is not being used. A plain HTML textarea is being used, hence Quill.js is not necessary.